docs/welcome/install.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Install ParadeDB
+---
+
+The fastest way to install ParadeDB is by pulling the ParadeDB docker image and running it locally.
+
+```bash
+docker run \
+  --name paradedb \
+  -e POSTGRES_USER=myuser \
+  -e POSTGRES_PASSWORD=mypassword \
+  -e POSTGRES_DB=mydatabase \
+  -v paradedb_data:/var/lib/postgresql/data/ \
+  -p 5432:5432 \
+  -d \
+  paradedb/paradedb:latest
+```
+
+You may replace `myuser`, `mypassword`, and `mydatabase` with whatever values you want — these will be your database
+connection credentials. By default, this will start a ParadeDB instance at `http://localhost:5432`.
+
+To connect to ParadeDB, install the `psql` client and run
+
+```bash
+psql -h <hostname> -U <user> -d <dbname> -p <port> -W
```

Comment:
I've observed this only works on macOS. On Linux, they need to preface it by `docker exec -it` for a shell that then runs this. We should update it to:

```bash
docker exec -it paradedb psql -U myuser -d mydatabase -W
```

The hostname can be omitted since it's `localhost` and so can the port since 5432 is the default. But we can include them for consistency.